css/style.css

```diff
@@ -1537,7 +1541,11 @@ li.skills {
   }
 
   #form .form-btn {
-    margin-left: 54% !important;
+    margin-left: 54%;
+  }
+
+  .contact-form form .email-msg {
```

Comment:
- [ ] You have two buttons you accidentally duplicated it
![hhh](https://user-images.githubusercontent.com/61562626/128387700-9ac35672-aa5f-49c1-9b89-027ff95e0baf.jpeg)